### PR TITLE
fix: Backwards dropcaps on the web

### PR DESCRIPTION
**Pull request by jmars**
Fixes the dropcaps being backwards on articles with markup dropcaps.
image

### CODE REVIEW

File: packages/link/src/link.web.js [hunk 1]
```diff
@@ -10,12 +10,19 @@ const RespLink = responsiveLinkStyles =>
...
+  target,
+  url,
+  underlined
+}) => {
   const Wrapper =
     responsiveLinkStyles !== null ? RespLink(responsiveLinkStyles) : "a";
 
   const style =
-    responsiveLinkStyles !== null
+    underlined && responsiveLinkStyles !== null
```
Comment (GaboDimitrov) [inline packages/link/src/link.web.js]: Do you need the **!== null** part here?

File: packages/article-skeleton/src/article-body/article-link.web.js [hunk 2]
```diff
@@ -19,9 +20,23 @@ const responsiveLinkStyles = {
...
 };
 
-const ArticleLink = ({ children, target, url, onPress }) => (
+const dropCapLinkStyles = {
+  base: `
+    font-size: inherit;
+    text-decoration: none;
+    color: ${colours.functional.action};
+  `,
+  medium: `
```
Comment (aronshamash) [inline packages/article-skeleton/src/article-body/article-link.web.js]: can we DRY this up with a common string? Does it need to be repeated for medium anyway ?

File: packages/article-skeleton/src/dropcap-util.web.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default (children, template, isDropcapDisabled) => {
   );
   if (newContent.length > 0 && newContent[0].name === "dropCap") {
     // remove the wrapping paragraph
-    newContent[0].children = newContent[0].children[0].children;
+    newContent[0].children = newContent[0].children[0].children.slice(0, 1);
```
Comment (aronshamash) [inline packages/article-skeleton/src/dropcap-util.web.js]: I was expecting a new test case as we needed to change the implementation. Is it not needed ?

Comment (parkjh) [reply packages/article-skeleton/src/dropcap-util.web.js]: @aronshamash are you still working on this?

**Pull request merged**